lib/global_config.js

```diff
@@ -180,41 +180,75 @@ exports.jsonColumnVariantParser = rawColumnValue => vm.runInContext("(" + rawCol
  *
  * @param {function: (rawColumnValue: string) => any} value
  */
-exports.setJsonColumnVariantParser = function (value)
-{
+exports.setJsonColumnVariantParser = function (value){
   // validate input
   Errors.assertInternal(Util.isFunction(value));
 
   exports.jsonColumnVariantParser = value;
 };
 
-// The default XML parser
-exports.xmlColumnVariantParser = rawColumnValue =>
-{
-  // check if raw string is in XML format
-  // ensure each tag is enclosed and all attributes and elements are valid
-  // XMLValidator.validate returns true if valid, returns an error if invalid
-  var validateResult = XMLValidator.validate(rawColumnValue);
-  if (validateResult === true)
-  {
-    // use XML parser
-    return new XMLParser().parse(rawColumnValue);
-  }
-  else
-  {
-    throw new Error(validateResult.err.msg);
-  }
+/**
+ * As a default we set parameters values identical like in fast-xml-parser lib defaults
+ * thus preserving backward compatibility if customer doesn't set custom configuration
+ */
+const defaultXmlParserConfiguration = {
+  ignoreAttributes: true,
+  alwaysCreateTextNode: false,
+  attributeNamePrefix: '@',
+  attributesGroupName: null
 };
 
+// The default XML parser
+exports.xmlColumnVariantParser = getXmlColumnVariantParser(defaultXmlParserConfiguration);
+
 /**
  * Updates the value of the 'xmlColumnVariantParser' parameter.
+ * Return custom XmlParser configuration or default if not exists.
  *
  * @param {function: (rawColumnValue: string) => any} value
  */
-exports.setXmlColumnVariantParser = function (value)
-{
+exports.setXmlColumnVariantParser = function (value){
   // validate input
   Errors.assertInternal(Util.isFunction(value));
 
   exports.xmlColumnVariantParser = value;
 };
+/**
+ * Create and update the 'xmlColumnVariantParser' parameter using custom parser configuration.
+ *
+ * @param {function: (rawColumnValue: string) => any} params
+ */
+exports.createXmlColumnVariantParserWithParameters = function (params){
```

Comment:
is it create or just configure?

Reply:
inside exists code ```new XmlParse(parserConfiguration)```
, that's why it is creation for me